mouseion/mouseion/maketokens.py

```diff
@@ -0,0 +1,168 @@
+from tasks import Task, TaskProcessor, ProgressType
+from typing import Dict
+from multiprocessing import Queue
+import orjson
+import os
+from pathlib import Path
+from transformers import T5Tokenizer
+import numpy as np
+from multiprocessing import Manager
+import torch
+from sentence_transformers import SentenceTransformer
+
+
+def get_tokenizer():
+    # Set model_max_length to suppress warning
+    #    return AutoTokenizer.from_pretrained("t5-base", use_fast=False, model_max_length=10000)
+    tokenizer = T5Tokenizer.from_pretrained("t5-small")
+    return tokenizer
+
+
+def function_load_jsonl(size: int, progress_queue: Queue, **kwargs) -> list:
+    file_path = "/mnt/datasets/arxiv/arxiv_trained_minimized.json"
+    split_count = kwargs['split_count']
+    file_size = os.path.getsize(file_path)
+    file = open(file_path)
+    progress_queue.put(10000000)
+
+    i = 0
+    splits = [[] for _ in range(32)]
+
+    for line in file:
```

Comment:
This could use `enumerate()` to ensure that `i` is bumped once per loop, saving two lines of code and providing a loop invariant.

Reply:
sounds fine. I didn't intend to add these to the token builder. I will remove them. I have a seperate pr I am wokring on now that properly builds an index. The current index is trained, but not built with index.add().

Reply:
this has been corrected.